patroni/ha.py

```diff
@@ -227,6 +231,10 @@ def touch_member(self):
             if self.is_paused():
                 data['pause'] = True
 
+            # Note time we last updated DCS
+            self.dcs_last_seen = int(time.time())
+            data['dcs_last_seen'] = self.dcs_last_seen
+
```

Comment:
This doesn't make much sense. If we can update the member information in DCS, it automatically means that we just have seen it.

Reply:
Right ok, done